### ISSUE TITLE
Line numbers are off when using skip_frontmatter: true

### ISSUE THREAD

**Issue by agross** [brigade/haml-lint#124]
I have `skip_frontmatter: true` in `.haml-lint.yml`.

Warning displayed in Atom:
> LineLength: Line is too long. [112/80] at line 9 col 5

```haml
---
title: Veranstaltungen
---
%h1
  - unless @item.fetch(:year, nil)
    Zukünftige
  Veranstaltungen
  - if @item.fetch(:year, nil)
    von                                  # Line 9 is reported as too long.
    = @item.fetch(:year)

- events = Events.new(@items).in_year(@item.fetch(:year, Date.today.year), future: @item.fetch(:year, nil).nil?)
# The last line is the actually offending line.
```

`haml-lint` also reports wrong line numbers:

```shell
$ haml-lint content/veranstaltungen.haml
content/veranstaltungen.haml:9 [W] LineLength: Line is too long. [112/80]
```

This affects editor plugins like the linters for Sublime Text and Atom.

I initially [reported the issue to Atom's linter-haml repository](https://github.com/AtomLinter/linter-haml/issues/42), but due to the fact that the shell command also shows line numbers off I think this is rather a core issue.
